alerting/src/main/kotlin/org/opensearch/alerting/BucketLevelMonitorRunner.kt

```diff
@@ -30,8 +30,13 @@ import java.time.Instant
 object BucketLevelMonitorRunner : MonitorRunner {
     private val logger = LogManager.getLogger(javaClass)
 
-    override suspend fun runMonitor(monitor: Monitor, monitorCtx: MonitorRunnerExecutionContext, periodStart: Instant, periodEnd: Instant, dryrun: Boolean):
-        MonitorRunResult<BucketLevelTriggerRunResult> {
+    override suspend fun runMonitor(
+        monitor: Monitor,
+        monitorCtx: MonitorRunnerExecutionContext,
+        periodStart: Instant,
+        periodEnd: Instant,
+        dryrun: Boolean
+    ): MonitorRunResult<BucketLevelTriggerRunResult> {
```

Comment:
thanks for refactoring. Wondering if this should be a common checkstyle check which should be available with the package for consistency during development.